Compilation failure with modern boost

```
src/sjef/util/Shell.h:3:10: fatal error: boost/process/child.hpp: No such file or directory
    3 | #include <boost/process/child.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
From the [release notes](https://www.boost.org/users/history/version_1_86_0.html) for version 1.86.0:
```
Moved the old boost.process to v1 subfolder and inline namespace. deprecated process/*.hpp v1 headers
```

--- a/pysjef/build.sh
+++ b/pysjef/build.sh
@@ -15,6 +15,7 @@ PREFIX=${PREFIX:-${CONDA_PREFIX}}
 BUILD=cmake-build
 echo PREFIX=${PREFIX}
 echo CONDA_PREFIX=${CONDA_PREFIX}
+conda list
 
 if [ -x "$(command -v $CONDA_PREFIX/bin/cc)" ]
 then
@@ -44,4 +45,4 @@ grep CMAKE_PROJECT_VERSION: $BUILD/CMakeCache.txt | sed -e 's/.*=/__version__ = 
 
 #PREFIX=$PREFIX python setup.py build_ext --inplace
 #PREFIX=$PREFIX python -m pip install --no-deps --force-reinstall --verbose . 
-PREFIX=$PREFIX python -m pip install -e . 
+PREFIX=$PREFIX python -m pip install .

--- a/src/sjef/util/Shell.cpp
+++ b/src/sjef/util/Shell.cpp
@@ -1,6 +1,11 @@
 #include "Shell.h"
+#if __has_include(<boost/process/child.hpp>)
 #include <boost/process/search_path.hpp>
 #include <boost/process/spawn.hpp>
+#else
+#include <boost/process/v1/search_path.hpp>
+#include <boost/process/v1/spawn.hpp>
+#endif
 #include <chrono>
 #include <filesystem>
 #include <regex>

--- a/src/sjef/util/Shell.h
+++ b/src/sjef/util/Shell.h
@@ -1,8 +1,13 @@
 #ifndef SJEF_LIB_UTIL_SHELL_H_
 #define SJEF_LIB_UTIL_SHELL_H_
 #include "Logger.h"
+#if __has_include(<boost/process/child.hpp>)
 #include <boost/process/child.hpp>
 #include <boost/process/io.hpp>
+#else
+#include <boost/process/v1/child.hpp>
+#include <boost/process/v1/io.hpp>
+#endif
 namespace bp = boost::process;
 
 namespace sjef::util {

--- a/src/sjef/util/Shell.h
+++ b/src/sjef/util/Shell.h
@@ -1,12 +1,12 @@
 #ifndef SJEF_LIB_UTIL_SHELL_H_
 #define SJEF_LIB_UTIL_SHELL_H_
 #include "Logger.h"
-#if __has_include(<boost/process/child.hpp>)
-#include <boost/process/child.hpp>
-#include <boost/process/io.hpp>
-#else
+#if __has_include(<boost/process/v1/child.hpp>)
 #include <boost/process/v1/child.hpp>
 #include <boost/process/v1/io.hpp>
+#else
+#include <boost/process/child.hpp>
+#include <boost/process/io.hpp>
 #endif
 namespace bp = boost::process;
 

--- a/test/test-sjef-molpro.cpp
+++ b/test/test-sjef-molpro.cpp
@@ -2,7 +2,11 @@
 #include <gtest/gtest.h>
 #include <signal.h>
 
+#if __has_include(<boost/process/search_path.hpp>)
 #include <boost/process/search_path.hpp>
+#else
+#include <boost/process/v1/search_path.hpp>
+#endif
 #include <chrono>
 #include <fstream>
 #ifndef WIN32

--- a/test/test-sjef-molpro.cpp
+++ b/test/test-sjef-molpro.cpp
@@ -2,10 +2,10 @@
 #include <gtest/gtest.h>
 #include <signal.h>
 
-#if __has_include(<boost/process/search_path.hpp>)
-#include <boost/process/search_path.hpp>
-#else
+#if __has_include(<boost/process/v1/search_path.hpp>)
 #include <boost/process/v1/search_path.hpp>
+#else
+#include <boost/process/search_path.hpp>
 #endif
 #include <chrono>
 #include <fstream>

